<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuggestionQuery</class>
 <widget class="QMainWindow" name="SuggestionQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SuggestionQuery</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(136, 202, 94);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="textBox1">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>190</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 192, 185);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_Key">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>q</string>
    </property>
   </widget>
   <widget class="QPushButton" name="w_Key">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>w</string>
    </property>
   </widget>
   <widget class="QPushButton" name="r_Key">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QPushButton" name="t_Key">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 151, 255);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>t</string>
    </property>
   </widget>
   <widget class="QPushButton" name="y_Key">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="e_Key">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="u_Key">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>u</string>
    </property>
   </widget>
   <widget class="QPushButton" name="i_Key">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
   <widget class="QPushButton" name="a_Key">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p_Key">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="o_Key">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>o</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s_Key">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="d_Key">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>d</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f_Key">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>f</string>
    </property>
   </widget>
   <widget class="QPushButton" name="g_Key">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>g</string>
    </property>
   </widget>
   <widget class="QPushButton" name="h_Key">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QPushButton" name="j_Key">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>j</string>
    </property>
   </widget>
   <widget class="QPushButton" name="k_Key">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>k</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l_Key">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>l</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_Key">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x_Key">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c_Key">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="v_Key">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_Key">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n_Key">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>n</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_Key">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_Key">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="space_Key">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Space</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter_Button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Sans Serif&quot;;
border-radius: 10px; 
background-color: rgb(255, 255, 255);
color: rgb(136, 202, 94);</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Sans Serif&quot;;
border-radius: 10px; 
background-color: rgb(255, 255, 255);
color: rgb(136, 202, 94);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Welcome to</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(241, 192, 185);
font: 75 24pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Suggestions</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Please enter a keyword describing your symptoms so I can &lt;br&gt;provide you suggestions based on your current conditions.</string>
    </property>
   </widget>
   <widget class="QListWidget" name="suggestionsListBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <zorder>textBox1</zorder>
   <zorder>q_Key</zorder>
   <zorder>w_Key</zorder>
   <zorder>r_Key</zorder>
   <zorder>t_Key</zorder>
   <zorder>y_Key</zorder>
   <zorder>e_Key</zorder>
   <zorder>u_Key</zorder>
   <zorder>i_Key</zorder>
   <zorder>a_Key</zorder>
   <zorder>p_Key</zorder>
   <zorder>o_Key</zorder>
   <zorder>s_Key</zorder>
   <zorder>d_Key</zorder>
   <zorder>f_Key</zorder>
   <zorder>g_Key</zorder>
   <zorder>h_Key</zorder>
   <zorder>j_Key</zorder>
   <zorder>k_Key</zorder>
   <zorder>l_Key</zorder>
   <zorder>z_Key</zorder>
   <zorder>x_Key</zorder>
   <zorder>c_Key</zorder>
   <zorder>v_Key</zorder>
   <zorder>b_Key</zorder>
   <zorder>n_Key</zorder>
   <zorder>m_Key</zorder>
   <zorder>back_Key</zorder>
   <zorder>space_Key</zorder>
   <zorder>enter_Button</zorder>
   <zorder>back_Button</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>suggestionsListBox</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
